docs/crc-guide.md

```diff
@@ -0,0 +1,162 @@
+# Running Clowder on CRC 
+
+## Prerequisites
+-----------------
+1. Download the [crc binary](https://developers.redhat.com/products/codeready-containers/overview) and follow the instructions to get crc running.
+2. Fork or clone the [Clowder repo](https://github.com/RedHatInsights/clowder)
+3. Install [the Clowder dependencies](https://github.com/RedHatInsights/clowder#dependencies)
+4. Run `make install`
+
+
+## Running Clowder
+-------------------
+At this time it is not recommended to deploy Clowder to your crc cluster. Instead, we will run the Operator on your local machine. To do this, we need add `ingress-env-minio.clowder-system.svc` to your `/etc/hosts` localhost (127.0.0.1). 
```

Comment:
May want to note that `ingress-env` is the name of your environment and that if they use anything else it will be different :)